### ISSUE TITLE
MELPA Stable not being updated

### ISSUE THREAD

**Issue by doublep** [melpa/melpa#7538]
From the site: *Next build: a month ago, last ended a month ago and took an hour*

I released Eldev 0.9 yesterday and today noticed it is still not available from MELPA Stable.

**Comment by redguardtoo** [comment]
I noticed too. The find-file-in-project at Melba stable is `6.0.3` which is publish at March 15. But flip current version is `6.0.6` published a few days ago.

**Comment by purcell** [comment]
Thanks folks. I've been a little suspicious about this recently too. I suspect that changes in `package-build` have caused a regression here. Maybe @tarsius has a guess? I don't think it's the "partial clone" changes I introduced, because those affected packages still have working directories that are full clones.

**Comment by purcell** [comment]
I suspect Makefile changes actually.

**Comment by purcell** [comment]
Yeah, I think I broke this in ab71444f97c23b334d5575213dbc5f6bf52eaa93 sorry. Will work on a fix.

**Comment by purcell** [comment]
So what happened was that it became possible for `make recipes/x` invocations to fail the build. The main build of everything *except* for `python-info` and `python3-info` was safeguarded so that it was allowed to fail without the containing script exiting. *However*, there was a separate build of those two specific recipes, and those were _not_ safeguarded. At some point, `python3-info` was removed from MELPA, so the failure at that point started to cause the entire "build melpa" script to fail. This script is invoked twice from our docker task, first for Unstable and then for Stable, so what happened was that the task got aborted half-way, with the result that Stable stopped getting run.

I've deployed a fix now and hopefully it should resolve itself within a few hours.
